airbyte-scheduler/persistence/build.gradle

```diff
@@ -13,5 +13,6 @@ dependencies {
     implementation project(':airbyte-protocol:models')
     implementation project(':airbyte-scheduler:models')
 
+    testImplementation "org.flywaydb:flyway-core:7.14.0"
```

Comment:
aye this is confusing.

this dependency is basically needed everywhere we call `jobMigrator.migrate()`. in this case, it's L166 - 169 of the `DefaultJobPersistenceTest` java file.

Reply:
This is needed because some of the jobs persistence unit tests need to run the migration first. When we launch a database, it only has the initial schema.